internal/resources/tanzupackageinstall/spec/cluster_scope.go

```diff
@@ -66,7 +67,7 @@ func ConstructSpecForClusterScope(d *schema.ResourceData) (spec *packageinstallm
 
 	if v, ok := specData[InlineValuesKey]; ok {
 		if v1, ok := v.(map[string]interface{}); ok {
-			spec.InlineValues = common.GetTypeStringMapData(v1)
+			spec.InlineValues = v1
```

Comment:
Why was this change needed?

Reply:
we don't want for inline values only string type of data would be there